### ISSUE TITLE
Should this work with a React app? TypeError: URL is not a constructor coming from cloud-config-client

### ISSUE THREAD

**Issue by dashaun** [MattRoss34/nice-config#1]
I've tried with spring-cloud-config and nice-config and I run into the same thing.

```
TypeError: URL is not a constructor
./node_modules/cloud-config-client/index.js
node_modules/cloud-config-client/index.js:11
   8 | const https = require('https')
   9 | const url = require('url')
  10 | const URL = url.URL
> 11 | const DEFAULT_URL = new URL('http://localhost:8888')
  12 | const Config = require('./lib/config')
  13 | 
```
node v10.15.2
npm v6.14.4

**Comment by MattRoss34** [comment]
@dashaun this library should work fine with any node application.  What version of `cloud-config-client` do you have installed in your package?  Can you provide your package-lock.json or yarn-lock.json?

**Comment by dashaun** [comment]
https://github.com/dashaun/io.dashaun.npm.spring-cloud-config-client

**Comment by MattRoss34** [comment]
@dashaun this is fixed in v1.0.3.

Happy configuring :)
